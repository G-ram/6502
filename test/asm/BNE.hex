a9 04 4c 0b 06 a9 07 8d 00 01 00 d0 f8 a9 08 8d
00 01 
