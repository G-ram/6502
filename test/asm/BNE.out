A: 07
X: 00
Y: 00
P: 00010000
S: 00
PC: 60a
100 : 07
101 : 00
